algorithms/brute-force.py

```diff
@@ -0,0 +1,152 @@
+# Sourced from https://github.com/jwngr/sdow
```

Comment:
You have to include the [text of the license](https://github.com/jwngr/sdow/blob/master/LICENSElicense) in any files that use code from the project.